components/footer.js

```diff
@@ -80,7 +80,7 @@ const handleThemeChange = (dark) => {
 
 const RssPopover = (
   <Popover>
-    <Popover.Content style={{ fontWeight: 500, fontSize: '.9rem' }}>
+    <Popover.Content className='d-flex' style={{ fontWeight: 500, fontSize: '.9rem' }}>
```

Comment:
This was needed to fix this:

![2023-06-12-120757_298x115_scrot](https://github.com/stackernews/stacker.news/assets/27162016/56dbe2cf-0ec2-4038-b6ef-6617b22514a1)

Now looks like this:

![2023-06-12-122651_290x91_scrot](https://github.com/stackernews/stacker.news/assets/27162016/b2e9baa0-1f6a-4d10-b499-c1bc7cb46a0a)